sound/soc/sof/intel/hda.c

```diff
@@ -414,6 +415,22 @@ void hda_dsp_dump_skl(struct snd_sof_dev *sdev, u32 flags)
 	}
 }
 
+/* dump the first 8 dwords representing the extended ROM status */
+static void hda_dsp_dump_ext_rom_status(struct snd_sof_dev *sdev)
+{
+	char msg[128];
+	int len = 0;
+	u32 value;
+	int i;
+
+	for (i = 0; i < HDA_EXT_ROM_STATUS_SIZE; i++) {
+		value = snd_sof_dsp_read(sdev, HDA_DSP_BAR, HDA_DSP_SRAM_REG_ROM_STATUS + i * 0x4);
+		len += snprintf(msg + len, sizeof(msg) - len, "0x%x ", value);
```

Comment:
Sorry to nitpick the space is still in the wrong place:
-    len += snprintf(msg + len, sizeof(msg) - len, "0x%x ", value);
+   len += snprintf(msg + len, sizeof(msg) - len, " 0x%x", value);

The initial space is added on L431

Reply:
ok done. I think Im done with making this patch more beautiful for now!